hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/metadata/SparkHoodieBackedTableMetadataWriter.java

```diff
@@ -154,7 +154,7 @@ protected void commit(HoodieData<HoodieRecord> hoodieDataRecords, String partiti
       metadataMetaClient.reloadActiveTimeline();
```

Comment:
reloadActiveTimelice called here so not necessary in ccleanIfNeceasry/

Reply:
`compactIfNecessary` may trigger compaction so the timeline changes.